cmd/root.go

```diff
@@ -41,6 +40,8 @@ const (
 	PluginsDirectoryPathFlag = "plugin-directory-path"
 	ConfigEncryptionKeyFlag  = "config-encryption-key"
 	ListenFlag               = "listen"
+	ListenProfilerFlag       = "listen-profiler"
+	EnableProfilerFlag       = "enable-profiler"
```

Comment:
Maybe add flags in go-libs ?

Reply:
here now: https://github.com/formancehq/go-libs/pull/196